docs/migration-guides/qiskit-runtime.mdx

```diff
@@ -53,14 +71,25 @@ See these topics for instructions:
 -  [Update code that calculates expectation values](qiskit-runtime-examples#estimator-algorithm)
 -  [Common use cases (basic, parameterized, and dynamic circuits)](qiskit-runtime-use-case)
 
-
-### Step 3a: replace any `backend.run` options with `qiskit_ibm_runtime` options.
+### Step 3a: Replace any `backend.run` options with `qiskit_ibm_runtime` options
 
 See the following topics for instructions:
 
 - [Migrate options](qiskit-runtime-options)
 - Common use cases [Options section](qiskit-runtime-use-case#migrate-options)
 
+### Step 3b: If you want to access a 3rd-party provider that only exposes `backend.run`, you can still follow a primitive-compatible your workflow using the Qiskit backend primitives
+
+See the following topic for instructions:
+
+- [Migrate using backend primitives](qiskit-backend-primitives)
+
+## Information to share with your users
+
+The following migration guides can help users with the following tasks:
+
+- [Transition to the Qiskit Runtime provider](/migration-guides/qiskit-runtime)
```

Comment:
Does this just link to the top of this same page?

Reply:
ok... but why is it called `Transition to the Qiskit Runtime provider`? Is it meant to say transitioning from `qiskit-ibmq-provider` / `qiskit-ibm-provider` to `qiskit-ibm-runtime` ?

Reply:
I'm not sure why it's called like this, but given that the title of the page is now "Migrate to primitives", and that this is self-referencing, I think this one would make more sense:

```suggestion
- [Transition from `backend.run` to primitives](/migration-guides/qiskit-runtime)
```

Reply:
I agree that your suggested title makes more sense.  Currently, the title of this file is actually "Overview", which is clearly a mistake, so I'll update it. You can verify that I named it correctly.

Reply:
Also, @abbycross told me that we're going to use backend.run without (), so I updated those in this file as well.